File: ws/chavez/u2/ws16Libraries/TaxesLib/src/utils/Tax.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package utils;

/**
 *
 * @author Odalys Chavez, CodeBreakers, @ESPE
 */
public class Tax {
    public static float computeIva(float amount, float taxPercentage){
        float taxValue;
        taxValue=amount * taxPercentage/100;
        return taxValue;   
    }
    
    public static float computeTotal(float amount, float taxPercentage){
        float totalValue;
        totalValue = amount + computeIva(amount, taxPercentage);
        return totalValue;
    }
    
}

File: hw/tipan/u2/hw16Libreries/JavaScript/utils/tax.js
class Tax {

    static computeIva(amount, taxPercentage) {

        let taxValue;

        taxValue = amount * taxPercentage / 100;

        return taxValue;
    }

    static computeTotal(amount, taxPercentage) {

        let totalValue;

        totalValue = amount + Tax.computeIva(amount, taxPercentage);

        return totalValue;
    }
}

module.exports = Tax;